tutorials/optimization/optimizing_3d_performance.rst

```diff
@@ -22,57 +22,64 @@ in the street you are in, as well as the sky and a few birds flying overhead. As
 far as a naive renderer is concerned however, you can still see the entire town.
 It won't just render the buildings in front of you, it will render the street
 behind that, with the people on that street, the buildings behind that. You
-quickly end up in situations where you are attempting to render 10x, or 100x
-more than what is visible.
+quickly end up in situations where you are attempting to render 10× or 100× more
+than what is visible.
 
 Things aren't quite as bad as they seem, because the Z-buffer usually allows the
-GPU to only fully shade the objects that are at the front. However, unneeded
-objects are still reducing performance.
+GPU to only fully shade the objects that are at the front. This is called *depth
+prepass* and is enabled by default in Godot when using the GLES3 renderer.
+However, unneeded objects are still reducing performance.
 
 One way we can potentially reduce the amount to be rendered is to take advantage
-of occlusion. As of version 3.2.2 there is no built in support for occlusion in
-Godot, however with careful design you can still get many of the advantages.
+of occlusion. As of Godot 3.2.2, there is no built in support for occlusion in
+Godot. However, with careful design you can still get many of the advantages.
 
-For instance in our city street scenario, you may be able to work out in advance
+For instance, in our city street scenario, you may be able to work out in advance
 that you can only see two other streets, ``B`` and ``C``, from street ``A``.
 Streets ``D`` to ``Z`` are hidden. In order to take advantage of occlusion, all
 you have to do is work out when your viewer is in street ``A`` (perhaps using
 Godot Areas), then you can hide the other streets.
 
-This is a manual version of what is known as a 'potentially visible set'. It is
+In some cases, you may have to adapt your level design to add more occlusion
+oppotunities. For example, you may have to add more walls to prevent the player
+from seeing too far away, which would decrease performance due to the lost
+opportunies for occlusion culling.
+
+This is a manual version of what is known as a "potentially visible set". It is
```

Comment:
This paragraph refers to the paragraph that used to be directly above and should be moved back up. The added paragraph above can go under this one just fine as it is application to more than just PVS